doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -1541,8 +1545,17 @@ could each be tagged with a consistent name to make finding logs easier.::
     sometimes avoid setting internal ``@`` headers for passing information between hooks.
     These internal state variables are much more efficient than setting and reading headers.
 
+Prepend a Path to the URL
+-------------------------
+
+The following prepends ``/foo/bar`` to the URL path. Note that since ``PATH`` is
+non-inclusive of the initial ``/``, the provided prefix does not start with a ``/``::
```

Comment:
This was confusing to me -- the literal string PATH in the example versus the concept of PATH. I think I would be less confused if we add "but, it does end with a trailing /". Eg, if the url is example.com/example/ then providing foo/bar/%{CLIENT-URL:PATH} will yield example.com/foo/bar/example/ (and foo/bar%{CLIENT-URL:PATH} will yield example.com/foo/barexample/)

Reply:
I think I improved things. I added an example similar to the one you provided here too. Let me know what you think and we can iterate.

Reply:
Like the doc updates!
Interestingly, this week we were troubleshooting a string concat problem, where I believe the solution was to remove the quotes (they yielded a Warning). The initial string was like
set-header X-Foo "%{HEADER:Cache-Control}, %{Age}"
... the warning went away with 
set-header X-Foo %{HEADER:Cache-Control}, %{Age}